index.md

```diff
@@ -549,12 +550,17 @@ DMS-1719818 and DMS-1943186.
 
 This project has benefited from funding from the German
 [Federal Ministry of Education and Research](https://www.bmbf.de) (BMBF)
-through the project grant "Adaptive earth system modeling
-with significantly reduced computation time for exascale supercomputers
-(ADAPTEX)" (funding id: 16ME0668K).
+through the following grants:
+* Project grant "Adaptive earth system modeling with significantly reduced computation time for
+  exascale supercomputers (ADAPTEX)" (funding id: 16ME0668K)
+* Project grant "XXX" (funding id: 01LK2315B).
```

Comment:
@amrueda Please fix

Reply:
Done in 09e297420610c68326dc40c1661e62e52fbbbddf.